### PR TITLE
LED Update

### PR DESCRIPTION
**Pull request by allencai11**

### CODE REVIEW

File: 2024 Code/src/main/java/frc/robot/commands/AutoAlignCommand.java [hunk 1]
```diff
@@ -26,6 +27,8 @@ public void execute() {
         currentYaw = SubsystemContainer.swerveSubsystem.getYaw();
         targetYaw = SubsystemContainer.limelightSubsystem.getTx();
         SubsystemContainer.swerveSubsystem.setRobotYaw(currentYaw - targetYaw);
+        LEDSubsystem.isRunning = false;
```
Comment (Anirudh4Neutrino) [inline 2024 Code/src/main/java/frc/robot/commands/AutoAlignCommand.java:30]: create getter and setter methods

**Review by Anirudh4Neutrino — APPROVED** [review]

File: 2024 Code/src/main/java/frc/robot/commands/AutoAlignCommand.java [hunk 2]
```diff
@@ -26,6 +26,7 @@ public void execute() {
         currentYaw = SubsystemContainer.swerveSubsystem.getYaw();
         targetYaw = SubsystemContainer.limelightSubsystem.getTx();
         SubsystemContainer.swerveSubsystem.setRobotYaw(currentYaw - targetYaw);
+        SubsystemContainer.swerveSubsystem.setCommandState("autoalign");
```
Comment (gurellia53) [inline 2024 Code/src/main/java/frc/robot/commands/AutoAlignCommand.java:29]: This is fine for now, but I think using an enum would be more appropriate. strings like this are relatively processor intensive.

Comment (allencai11) [reply 2024 Code/src/main/java/frc/robot/commands/AutoAlignCommand.java:29]: Ok sounds good. I'll start working on that. Can you explain how an enum works?

Comment (gurellia53) [reply 2024 Code/src/main/java/frc/robot/commands/AutoAlignCommand.java:29]: https://stackoverflow.com/questions/9850525/whats-the-use-of-enum-in-java

**Pull request merged**